src/js/utils/DOMUtils.js

```diff
@@ -28,21 +29,19 @@ var DOMUtils = {
 
     var computeInnerBound = function (acc, key) {
       var val = parseInt(compstyle[key], 10);
-      if (_.isNaN(val)) {
+      if (Number.isNaN(val)) {
```

Comment:
fear not all our fears can be laid to rest: 
![](http://cl.ly/340z220E0l03/Image%202016-05-03%20at%2011.40.21%20PM.png)

(reference: https://kangax.github.io/compat-table/es6/#test-Number_properties)